Header Header
uint8 arm       # 0 = left, 1 = right

# all values in radians
float64 shoulder_flappy
float64 shoulder_foraft
float64 upperarm_rotate
float64 elbow
float64 wrist_rotate
float64 wrist_pan
float64 wrist_tilt
